Scripts/AccessdataCheckProcessExistsInSnapshot/AccessdataCheckProcessExistsInSnapshot.py

```diff
@@ -0,0 +1,26 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+# Constant and mandatory arguments
+file_path = demisto.get(demisto.context(), 'Accessdata.Job.Result.SnapshotDetails.File')
```

Comment:
instead loading this value from context like global object, which is a bad practice, declare an input argument for this script and pass it in the playbook as an argument